gateways/authorizenet/request_builders.go

```diff
@@ -134,21 +138,25 @@ func addL2L3Data(authRequest *sleet.AuthorizationRequest, authNetAuthRequest *Cr
 			Amount: strconv.FormatInt(authRequest.Level3Data.ShippingAmount.Amount, 10),
 		}
 
-		authNetAuthRequest.TransactionRequest.Customer = &Customer{
-			Id: authRequest.Level3Data.CustomerReference,
+		if authNetAuthRequest.TransactionRequest.Customer != nil {
```

Comment:
Curious: when would authNetAuthRequest.TransactionRequest.Customer != nil? Don't we usually check with `authRequest` to build `authNetAuthRequest`?

Reply:
ohh, this is paired with `Email`.